.erb-lint.yml

```diff
@@ -28,8 +28,8 @@ linters:
           - 'order-([0-3]|last)'
           - 'not-rounded'
           - 'rounded-(top|right|bottom|left)'
-          - '(md|lg)-hide'
-          - '(xs|md|lg)-show'
```

Comment:
Re: Removing `xs` here: I may have been confused in the original technical discovery, since BassCSS at some point in time had `xs-` prefixes:

https://github.com/basscss/basscss/blob/8104041/modules/hide/index.css#L12

But these do not exist in the NPM module version we're using:

https://github.com/basscss/basscss-sass/blob/v3.0.0/_responsive-states.scss#L66-L105